src/mantainer.rs

```diff
@@ -48,7 +48,7 @@ impl TableMantainer {
         outbound_sender: Sender<MessageBeanOut>,
     ) {
         debug!("TableMantainer::monitor_buckets started");
-        let idle_time: Duration = ktable.read().await.config.bucket_ttl;
+        let idle_time: Duration = { ktable.read().await.config.bucket_ttl };
         loop {
```

Comment:
If the reason to add an additional scope is to drop the lock, there should be a comment for it. Actually there should be a comment anyway, since the code is not clear